debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/LogicalDecodingMessageMonitor.java

```diff
@@ -99,6 +101,9 @@ public void logicalDecodingMessageEvent(Partition partition, OffsetContext offse
         value.put(DEBEZIUM_LOGICAL_DECODING_MESSAGE_KEY, logicalMsgStruct);
         value.put(Envelope.FieldName.SOURCE, offsetContext.getSourceInfo());
 
+        TableId tableId = PostgresSchema.parse("fake");
```

Comment:
perhaps this should be changed somehow

Reply:
Definitely not fake :-). Ideally (if possible) a new implementation of `io.debezium.spi.schema.DataCollectionId` will be defined in this class and it will either return empty string or strin like 'LOGICAL_MESSAGE' whatver will be feasible.